packages/legend-graph/src/graph-manager/__tests__/ImportResolutionRoundtrip.test.ts

```diff
@@ -249,3 +250,14 @@ describe(unitTest('DSL Data import resolution roundtrip'), () => {
     },
   );
 });
+
+describe(unitTest('SubTypeGrpahFetchTree roundtrip'), () => {
```

Comment:
why don't we add this test under ../legend-manual-tests/src/__tests__/roundtrip-grammar/cases ?